README.md

```diff
@@ -133,6 +141,167 @@ The health of the Elasticsearch container and / or cluster can be inspected by i
       "_source" : { "name": "John Doe" }
     }`
 
+7. Navigate to the Audit Web Service UI  
+    The Audit Web Service is a RESTful Web Service and is primarily intended for programmatic access, however it also ships with a Swagger-generated user-interface.
+
+    Using a browser, navigate to the `/caf-audit-service-ui` endpoint on the Audit Web Service:  
+
+        http://docker-host:25080/caf-audit-service-ui
```

Comment:
http://docker-host:25080/caf-audit-service-ui

Not sure if this is a recognised convention put I would normally place items like docker-host, which are to be substituted inside angle brackets and in caps, i.e. <DOCKER_HOST>

This also maintains the look and feel of the rest of the document.